pkg/types/ovirt/metadata.go

```diff
@@ -0,0 +1,12 @@
+package ovirt
+
+// Metadata contains ovirt metadata (e.g. for uninstalling the cluster).
+type Metadata struct {
```

Comment:
the metadata should only contain information required for destroying the cluster only. drop eveything that is not required for that purpose.